app/src/main/java/org/mozilla/firefox/vpn/main/MainActivity.kt

```diff
@@ -66,26 +64,26 @@ class MainActivity : AppCompatActivity() {
             finish()
         })
 
-        force_update.apply {
+        binding.forceUpdate.apply {
```

Comment:
Please make sure to test this scenario since force update is very difficult to test from QA side

Reply:
test OK 👍